bravado/response.py

```diff
@@ -55,3 +60,20 @@ def headers(self):
     @property
     def is_fallback_result(self):
         return self.exc_info is not None
+
+
+def get_metadata_class(fully_qualified_class_str):
+    class_to_import = _import_class(fully_qualified_class_str)
+    if not class_to_import:
+        return BravadoResponseMetadata
+
+    if issubclass(class_to_import, BravadoResponseMetadata):
+        return class_to_import
+
+    log.warning(
+        'bravado configuration error: the metadata class {fully_qualified_class_str}\' does not extend '
```

Comment:
nit: potentially you forgot the opening quote :)
Small note is about logging, please do not format the string yourself but use `%s` and params to logging method (ie. `log.warning('..the metadata class \'%s\' does not extend ...', fully_qualified_class_str)`). In this way the string will be built only if the content has to be logged